plugin-server/tests/main/db.test.ts

```diff
@@ -577,11 +578,60 @@ describe('DB', () => {
                 [2, 0, 'group_key', JSON.stringify({ prop: 'val-that-isnt-cached' })],
                 'testGroupPropertiesOnEvents'
             )
-            const res = await db.getGroupProperties(2, [{ index: 0, key: 'group_key' }])
+            const res = await db.getPropertiesForGroups(2, [{ index: 0, key: 'group_key' }])
             expect(res).toEqual({
                 group0_properties: '{"prop":"val","num":1234567}',
             })
         })
+
+        it('Gets created_at from DB if cache does not exist', async () => {
+            jest.spyOn(db, 'getGroupsCreatedAtFromDbAndUpdateCache')
+            // No cache exists as this was inserted directly
+            await db.postgresQuery(
```

Comment:
We have db.insertGroup - let's use that instead of rolling our own that we need to then maintain.

Reply:
The reason other tests also use this approach is that `insertGroup` caches the results, whereas here we want to explicitly bypass this

Reply:
However, I can add a cache option to that function